tst/com/amazon/ion/benchmark/DataGeneratorTest.java

```diff
@@ -186,73 +128,13 @@ public void testGeneratedFormat() throws Exception {
         }
     }
 
-    /**
-     * Assert the unicode code point range of the character which constructed the generated Ion string is conform with the expect range.
-     * @throws Exception if error occurs when executing Ion data generator.
-     */
-    @Test
-    public void testGeneratedStringUniCodeRange() throws Exception {
-        Map<String, Object> optionsMap = Main.parseArguments("generate", "--data-size", "500", "--data-type", "string", "--text-code-point-range", "[96,99]","test5.10n");
-        try (IonReader reader = DataGeneratorTest.executeAndRead(optionsMap)) {
-            List<Integer> range = WriteRandomIonValues.parseRange(optionsMap.get("--text-code-point-range").toString());
-            while (reader.next() != null) {
-                String str = reader.stringValue();
-                for (int i = 0; i < str.length(); i++) {
-                    int codePoint = Character.codePointAt(str, i);
-                    int charCount = Character.charCount(codePoint);
-                    //UTF characters may use more than 1 char to be represented
-                    if (charCount == 2) {
-                        i++;
-                    }
-                    assertTrue(codePoint >= range.get(0) && codePoint <= range.get(1));
-                }
-            }
-        }
-    }
-
-    /**
-     * Assert the generated timestamps is follow the precision and proportion of the given timestamp template.
-     * @throws Exception if error occurs when executing Ion data generator.
-     */
-    @Test
-    public void testGeneratedTimestampTemplateFormat() throws Exception{
```

Comment:
Do we have tests for generating timestamps from schema?

Reply:
> Do we have tests for generating timestamps from schema?
Yes, this test method is used for testing the generated timestamp from schema.
```
    @Test
    public void testViolationOfTimestamp() throws Exception {
        DataGeneratorTest.violationDetect(INPUT_ION_TIMESTAMP_FILE_PATH);
    }
```